micros/modules/system/boot/stage-1.nix

```diff
@@ -167,7 +167,7 @@
 
     replacements = {
       shell = "${extraUtils}/bin/ash";
-
+      modprobePkg = pkgs.kmod;
```

Comment:
you probably want to do something like:

```nix
modprobe = pkgs.modprobe + /bin/modprobe
```

and replace `@modprobe@` with that. Otherwise would be forcing users to always provide an executable called "modprobe", which can be slightly annoying to work around (`writeShellScriptBin`, etc.)